tests_metricflow/plan_conversion/test_time_spine.py

```diff
@@ -14,7 +15,9 @@ def test_date_spine_date_range(  # noqa: D103
     simple_semantic_manifest_lookup: SemanticManifestLookup,
     create_source_tables: None,
 ) -> None:
-    time_spine_source = TimeSpineSource.create_from_manifest(simple_semantic_manifest_lookup.semantic_manifest)
+    time_spine_source = TimeSpineSource.create_from_manifest(simple_semantic_manifest_lookup.semantic_manifest)[
+        TimeGranularity.DAY
```

Comment:
Oh ok. Technically this is the legacy time spine fallback.

Reply:
Yeah, honestly, I'm a bit confused about why this test is here. I guess it's essentially testing that the legacy time spine model we have documented somewhere has the values we expect. But this feels like it should live in core or something, not here, since it's not actually related to MF functionality.

Reply:
Actually it looks like its just testing the test data we have in `tests_metricflow/fixtures/source_table_snapshots/time_spine_table/mf_time_spine.yaml`! I guess that might be useful to keep people from changing it? ¯\_(ツ)_/¯